tic-tac-toe-app/Square.jsx

```diff
@@ -0,0 +1,31 @@
+const declareWinner = (states) => {
```

Comment:
Good idea here to create the `declareWinner()` outside of the scope of the `<Square/>` component.